dataflow-migrate-schedules/pom.xml

```diff
@@ -6,18 +6,18 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.1.RELEASE</version>
+		<version>2.2.6.RELEASE</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>io.spring</groupId>
 	<artifactId>migrateschedule</artifactId>
 	<version>1.0.0.BUILD-SNAPSHOT</version>
-	<name>Schedule Migrator</name>
-	<description>Migrates SCDF Schedules to the 2.3 format</description>
+	<name>Schedule CD Migrator</name>
+	<description>Migrates SCDF Schedules so that SchedulerTaskLauncher is no longer required.</description>
 
 	<properties>
 		<java.version>1.8</java.version>
-		<spring-cloud-data-flow.version>2.3.0.RC1</spring-cloud-data-flow.version>
+		<spring-cloud-data-flow.version>2.5.0.BUILD-SNAPSHOT</spring-cloud-data-flow.version>
 		<deployer.version>2.1.0.RELEASE</deployer.version>
```

Comment:
The deployer.version needs to be changed to use the latest `2.3.0.BUILD-SNAPSHOT`.